homeassistant/components/energy/data.py

```diff
@@ -49,10 +49,12 @@ class EnergyHomeConsumption(TypedDict):
 
     # This is an ever increasing value
     stat_consumption: str
+    entity_consumption: str | None
```

Comment:
The `statistic_id` will be identical to `entity_id` if the statistic is generated from a sensor, maybe we don't need both?
What do we do if `entity_id` is changed?

Reply:
Dealing with entity_id changes is out of scope. 

Yeah, we can drop the stat.

Reply:
But we still need to keep the stat as people can configure a statistic directly when it's sourced from another integration.